packages/ui/src/components/shared/settings/VerifyingPassword.tsx

```diff
@@ -7,7 +7,15 @@ import { useWalletState } from 'providers/WalletStateProvider';
 import { settingsDialogActions } from 'redux/slices/settings-dialog';
 import { Props } from 'types';
 
-const VerifyingPassword: FC<Props> = () => {
+interface VerifyingPasswordProps extends Props {
+  backButtonLabel?: string;
+  continueButtonLabel?: string;
+}
+
+const VerifyingPassword: FC<VerifyingPasswordProps> = ({
+  backButtonLabel = 'Back',
```

Comment:
We probably don't need to have option to customize the back button label, since most of the case it should always be `Back`